src/consensus/tx_verify.cpp

```diff
@@ -14,7 +14,13 @@
 #include <util/check.h>
 #include <util/moneystr.h>
 
-bool IsFinalTx(const CTransaction &tx, int nBlockHeight, int64_t nBlockTime)
+#include <algorithm>
+#include <cassert>
+#include <cstdint>
+#include <utility>
+#include <vector>
```

Comment:
I don't think we should be fixing up includes manually and review it manually, unless there is a reproducible bug.

I think, either this is done with iwyu in the CI task or not at all.

Reply:
Re-pushed with the changes suggested by iwyu.